pkg/providers/openstack/README.md

```diff
@@ -17,10 +17,11 @@ Start by selecting a unique name that will be used for the deployment's name, pr
 ```bash
 export USER=unikorn-staging
 export DOMAIN=unikorn-staging
+export PROJECT=unikorn-default
```

Comment:
Just have to be different don't you 😸  I'm going to change this so that you "specify a unique domain name on the platform", then that will get used to populate the project and user, just to simplify things further.  So you get let off here.